dbt_semantic_interfaces/validations/metrics.py

```diff
@@ -662,6 +662,33 @@ def validate_manifest(semantic_manifest: SemanticManifestT) -> Sequence[Validati
         return issues
 
 
+class MetricAggregationParamsAreOnlyForSimpleMetricsRule(
+    SemanticManifestValidationRule[SemanticManifestT], Generic[SemanticManifestT]
+):
+    """Checks that metric aggregation params are only set for simple metrics."""
+
+    @staticmethod
+    @validate_safely(
+        whats_being_done="running model validation ensuring metric aggregation params are only set for simple metrics"
+    )
+    def validate_manifest(semantic_manifest: SemanticManifestT) -> Sequence[ValidationIssue]:  # noqa: D
+        issues: List[ValidationIssue] = []
+
+        for metric in semantic_manifest.metrics or []:
+            if metric.type != MetricType.SIMPLE and metric.type_params.metric_aggregation_params is not None:
+                issues.append(
+                    ValidationError(
+                        context=MetricContext(
+                            file_context=FileContext.from_metadata(metadata=metric.metadata),
+                            metric=MetricModelReference(metric_name=metric.name),
+                        ),
+                        message=f"Metric '{metric.name}' is not a Simple metric, so it cannot have values for "
```

Comment:
This will be shown at the YAML validation level, so the user won't know what `metric_aggregation_params` are. Can we instead list each of the `metric_aggregation_params` that are set? Or even a generic list of all the attributes within `metric_aggregation_params` is fine too.